agency/README.md

```diff
@@ -173,8 +173,9 @@ Body Params:
 | ----------- | ----------------------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
 | `event_type` | Enum                         | Required          | see [Vehicle Events](#vehicle-events)                                                                                                           |
 | `event_type_reason` | Enum                  | Required if Available | see [Vehicle Events](#vehicle-events)                                                                                                           |
-| `timestamp`  | Timestamp                    | Required          | Date of last event update                                                     |
-| `telemetry`  | [Telemetry](#telemetry-data) | Required          | Single point of telemetry                             |
+| `timestamp`  | Timestamp                    | Optional          | Date of last event update                                                     |
```

Comment:
I believe `timestamp` is suppose to be `Required` and `telemetry` is suppose to be `Optional`?